Classes/Controller/NewsController.php

```diff
@@ -2,6 +2,12 @@
 
 namespace GeorgRinger\Eventnews\Controller;
 
+use GeorgRinger\Eventnews\Domain\Repository\LocationRepository;
+use GeorgRinger\News\Domain\Repository\CategoryRepository;
+use GeorgRinger\News\Domain\Repository\NewsRepository;
+use GeorgRinger\News\Domain\Repository\TagRepository;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
```

Comment:
@stigfaerch, this contradicts Georgs hint in https://github.com/georgringer/news/blob/master/Documentation/DeveloperManual/ExtendNews/ProxyClassGenerator/Index.rst

> Don't use any use statements as those are currently ignored!